drivers/video/video.c

```diff
@@ -3200,6 +3198,22 @@ static int video_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
   return ret;
 }
 
+static int video_mmap(FAR struct file *filep, FAR void **addr, off_t start,
+                      size_t length)
+{
+  FAR struct inode *inode = filep->f_inode;
+  FAR video_mng_t  *priv  = (FAR video_mng_t *)inode->i_private;
+  int ret = -EINVAL;
+
+  if (addr)
+    {
+      *addr = priv->video_inf.bufheap;
```

Comment:
add start to bufheap

Reply:
This is the original:
```
-      case FIOC_MMAP:
-        DEBUGASSERT((FAR void **)(uintptr_t)arg != NULL);
-        *(FAR void **)((uintptr_t)arg) = priv->video_inf.bufheap;
-        ret = OK;
-
-        break;
```
So not changing the functionality here..

Reply:
The change just make the driver handle nonzero offset correctly.

Reply:
Ok, will add